pmd-core/src/main/java/net/sourceforge/pmd/lang/ast/AntlrNode.java

```diff
@@ -46,11 +46,6 @@ default int jjtGetId() {
         throw new UnsupportedOperationException("Out of scope for antlr current implementations");
     }
 
-    @Override
-    default String getImage() {
-        throw new UnsupportedOperationException("Out of scope for antlr current implementations");
```

Comment:
You should return null here instead. Null is an acceptable default value for the image attribute.

Reply:
If we don't return the text content, why do we need to support getImage method?